eng/helix/helix.proj

```diff
@@ -79,9 +78,12 @@
     <HelixCorrelationPayload Include="$(HelixTestConfigurationFilePath)" AsArchive="false" />
   </ItemGroup>
 
-  <Target Name="IncludeAspNetRuntime" BeforeTargets="Gather"
-    Condition="'$(DoNotRequireSharedFxHelix)' != 'true' OR
-    EXISTS('$(RepoRoot)artifacts\packages\$(Configuration)\Shipping\Microsoft.AspNetCore.App.Runtime.$(TargetRuntimeIdentifier).$(SharedFxVersion).nupkg')">
```

Comment:
@BrennanConroy I realize this line didn't work despite my insistence on including it in your `$(DoNotRequireSharedFxHelix)` work. `$(SharedFxVersion)` was never set which meant RunHelix.ps1 couldn't successfully run App.Unitest or the project template tests ☹️